packages/config/src/api/site_info.js

```diff
@@ -71,3 +79,25 @@ const getAddons = async function (api, siteId) {
     throwUserError(`Failed retrieving addons for site ${siteId}: ${error.message}. ${ERROR_CALL_TO_ACTION}`)
   }
 }
+
+const getIntegrations = async function ({ api, ownerType, ownerId, logs }) {
+  if (ownerId === undefined) {
+    return []
+  }
+
+  try {
+    const token = api.accessToken()
+    const response = await fetch(`https://api.netlifysdk.com/${ownerType}/${ownerId}/integrations`, {
+      headers: {
+        Authorization: `Bearer ${token}`,
+      },
+    })
+
+    const integrations = await response.json()
+    return Array.isArray(integrations) ? integrations : []
+  } catch (error) {
+    logWarning(logs, `Failed retrieving integrations for ${ownerType} ${ownerId}: ${error.message}.`)
```

Comment:
This will emit a log to the user-facing build logs. What I suggested was for you to use our system logger, which pipes data to our internal logging infrastructure only.

You can find an example of it being used [here](https://github.com/netlify/build/blob/418914dddd7ddb888b5cd16a281df6bc1e856f68/packages/build/src/log/messages/core.ts#L39).

Reply:
sweet, cheers

Reply:
hmmm, `config` doesn't seem to have an inherent understanding of that currently, will look tomorrow at a nice way of bringing that in

Reply:
but, with it behind a featureflag, only we're using it for now, so no harm

Reply:
Yeah, just trying to avoid a situation where we forget to change this before you roll out the flag and then customers see an error that they can't self-resolve, which creates confusion and can lead to support escalations.

I'd suggest removing this until you're able to add a system log call, but happy to defer to you.

Reply:
I agree with that chief